docs/source/configuration/commandline.rst

```diff
@@ -82,8 +82,10 @@ Processing
 Additional data
 ~~~~~~~~~~~~~~~
 [-meta fileOrPath]
-    Single schema analysis file path to SchemaMeta-xml, when running ``-all`` or ``-schemas`` path to directory containing
-    SchemaMeta-xmls with pattern (DatabaseName|Schema).meta.xml
+    Single schema analysis file path to one (1) SchemaMeta-xml-file;
+    when running ``-all`` or ``-schemas`` it defines the path to directory containing one ore more SchemaMeta-xml-files with pattern (DatabaseName|Schema).meta.xml
```

Comment:
ore should probably or.

When analyzing single schema, should be path to a single schemameta <- (could insert link here)
When analyzing multiple schemas (``-all`` or ``-schemas``), should be path to a folder containing schemameta files with naming pattern [databaseName or schemaName].meta.xml I.e. You have schemas backoffice and frontoffice in your database to augment these with schemameta you would need to specify a folder as ``-meta`` and that folder should contain backoffice.meta.xml and frontoffice.meta.xml

Reply:
I used trial and error and after getting some errors.
That's why I think it could be explained better to get less errors.

Finally this worked fine and I added this example:

```
# one schema
schemaspy.s=config
# one path
schemaspy.meta=SchemaMeta
```

and I think this will work also with `-schemas` and `-all`

You can modify my request and improve it :-)

Reply:
Actually I think it's a weird implementation to allow file or folder. It would be much simpler to just say folder and it should contain file(s) named [DatabaseName or SchemaName].meta.xml, just allow one single use case.

Hmmm, even if it supports specifying a file, we could just document the folder usecase.....

Reply:
I agree, and if there is only one file it can also be in the folder. And if someone uses the file, then this will be a hidden feature.
Anyway may be this will change a little bit with 7.0 if the meta definition will be extended?